webapp/javascript/components/TimelineFlotChartSelection.ts

```diff
@@ -0,0 +1,298 @@
+/* eslint-disable */
+// @ts-nocheck
```

Comment:
How difficult would it be to add types to this plugin?

Reply:
this file was deleted